corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -274,3 +274,9 @@
     padding-left: 0px !important;
   }
 }
+
+.question-tile-row .gr-no-children.gr-has-no-questions.gr-has-no-nested-questions {
+  // Prevents a priority issue that's happened three times where
+  // a group with hidden values creates whitespace.
+  height: 0px !important;
```

Comment:
I believe this covers cases if margins/padding is added to its descendants. Should we use `display: none` so it covers cases where the `gr` class element has margin added to it?

Reply:
nevermind, you did this in a later commit